staticd/static_zebra.c

```diff
@@ -182,6 +182,8 @@ static void zebra_connected(struct zclient *zclient)
 	zclient_send_reg_requests(zclient, VRF_DEFAULT);
 
 	static_fixup_vrf_ids(vrf_info_lookup(VRF_DEFAULT));
+
+	bfd_nht_zclient_connected(zclient);
```

Comment:
We should check if BFD integration is initialized here as well (or move these checks to inside the API functions).

Reply:
bfd_nht_zclient_connected is removed in the second commit.